src/Page/Events/Event_.elm

```diff
@@ -142,8 +143,7 @@ viewInfoSection event =
                     text ""
             ]
         , div [ css [ eventDescriptionStyle ] ]
-            --  (Theme.TransMarkdown.markdownToHtml event.description)
```

Comment:
Did you check why this had previously been commented out?

Reply:
For styling, apparently. Looked like a temporary change that didn't feel intentional.